src/components/NumberInput.tsx

```diff
@@ -0,0 +1,15 @@
+import { useId } from "react";
+
+const NumberInput = () => {
+
+    const inputId = useId();
+
+    return (
+        <div className="number-input-container">
+            <label htmlFor={inputId} className="number-input-label">Please, enter number of questions!</label>
+            <input id={inputId} className="form-control" placeholder="enter number of questions from 5 to 15" />
+        </div>
+    );
+};
```

Comment:
how about refactoring this component a bit and making it more generic?
so, here you've hardcoded classNames, placeholders, etc.
I recommend you convert them into props and pass them down from the parent
besides that, you named this component as a `NumberInput` but you didn't define a type, so, by default input uses a `text` type tho